config/default.yml

```diff
@@ -205,6 +205,11 @@ Performance/StringReplacement:
   Enabled: true
   VersionAdded: '0.33'
 
+Performance/SubclassCheck:
+  Description: 'Use `A <= B` instead of `A.ancestors.include?(B)`.'
```

Comment:
Can you add `Reference`?
c.f. https://github.com/rubocop-hq/rubocop-performance/blob/v1.6.0/config/default.yml#L27